Sources/ACActionCable/ACSubscription.swift

```diff
@@ -26,14 +26,14 @@ public class ACSubscription {
     
     // MARK: Sending
     
-    public func send<T: Encodable>(object: T, completion: ACEventHandler? = nil) {
-        guard let command = ACCommand(type: .message, identifier: channelIdentifier, object: object) else { return }
+    public func send<T: Encodable>(action: String? = nil, object: T, completion: ACEventHandler? = nil) {
+        guard let command = ACCommand(type: .message, identifier: channelIdentifier, action: action, object: object) else { return }
         client.send(command, completion: completion)
     }
     
     public func send(action: String, completion: ACEventHandler? = nil) {
         guard let command = ACCommand(type: .message, identifier: channelIdentifier, action: action) else { return }
-        client.send(command)
+        client.send(command, completion: completion)
```

Comment:
Does this fix a previously existing bug? If so, please separate it out into its own commit for emphasis and add a test to prevent any future regression

Reply:
I would not say that it fixes a bug, but it was simply an oversight that the completion handler was not called properly